portfolio/src/main/webapp/index.html

```diff
@@ -5,9 +5,10 @@
     <title>Lilian Hunt</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAX-57oSyGZq4x4I4kgkHm7pUI0q2J-B30"></script>
```

Comment:
Please avoid putting the API key in github.

Use git-secrets (recommended) or a .gitignore file to exclude the file containing the service account credentials.(i.e. including /path/to/api_key.json in the .gitignore file)
Place the file with the API key outside of the Github project and retrieve it by creating an environment variable (i.e. GCP_API_KEY=/path/to/api_key.json).

Reply:
Please respond to the comment next time after addressing it.